packages/eslint-plugin/README.md

```diff
@@ -47,7 +47,10 @@ You can also enable all the recommended rules at once. Add `plugin:@typescript-e
 
 ```json
```

Comment:
you can use `cjson` instead to add comments to the json which get highlighted properly

Reply:
I struggle to add it to the example in a way that I would entirely like. How about noting eslint:recommended separately, in a paragraph of text? Or if you have an edit in mind that you'd like, feel free to add it.

Reply:
Updated in 86ae914a77adc48af7065ff85f426816fc90cc12.